lib/todo_lint/file_finder.rb

```diff
@@ -18,9 +20,11 @@ def initialize(path)
     # @api public
     # @return [Array<String>]
     def list(*extensions)
-      all_files.keep_if do |filename|
+      @all_files.keep_if do |filename|
```

Comment:
see if we can avoid referencing instance variables directly; maybe we can instead use the getter method provided by `attr_reader :all_files` (feel free to add more `attr_reader`s)